packages/iotex-react-block-producers/src/style-media.tsx

```diff
@@ -0,0 +1,74 @@
+export const PALM_WIDTH = 575;

```

Comment:
why add to iotex-react-block-producers?

if there are shared styles, please add to a new package packages/iotex-react-common

Reply:
i just delete unused comonent.
i think can add a new package `iotex-react-common`, can put some of  `src/shared/common` in this package,.
if can, i will open a new pr do this.